### PR TITLE
Preparing release 5.2.0

### PR DESCRIPTION
**Pull request by araratthehero**
## Description
[//]: # (Include a short summary of your changes)
[//]: # (If this is a new feature: attach screenshots or a video if applicable)
[//]: # (If this is a bug fix: include a reproduction path)
Preparing release 5.2.0: Updating version number and release notes.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -9,12 +9,22 @@
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
 ## New
+- We added a [UI customization guide](https://github.com/Adyen/adyen-android/blob/develop/docs/UI_CUSTOMIZATION.md) for the Android SDK.
```
Comment (jreij) [inline RELEASE_NOTES.md:12]: Can you use a relative path for the link instead of a full path?

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -9,12 +9,22 @@
...
 
+## Improved
+- The SDK now uses JSON Web Encryption (JWE) with RSA OAEP 256 and AES GCM 256 for encryption. You do not need to make any changes to your integration.
```
Comment (jreij) [inline RELEASE_NOTES.md:15]: Nit: I'm not sure if we usually refer to it as the "SDK" or "library", worth double checking?

Comment (araratthehero) [reply RELEASE_NOTES.md:15]: The technical writer suggested to use `The integration`. How does that sound?

File: RELEASE_NOTES.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (araratthehero) [reply RELEASE_NOTES.md:12]: Okay I can make it `We added a UI customization guide, which explains how to customize the styles and string resources.`

File: RELEASE_NOTES.md (hunk 2 above)
Comment (jreij) [reply RELEASE_NOTES.md:15]: 👌

File: RELEASE_NOTES.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jreij) [reply RELEASE_NOTES.md:12]: 👍

**Pull request merged**